app/jobs/notify_expired_rents_job.rb

```diff
@@ -0,0 +1,17 @@
+class NotifyExpiredRentsJob
+  include Sidekiq::Worker
+
+  def perform
+    Rent.ending_today.find_in_batches do |rent_batch|
```

Comment:
This is a very goob pratice 👏 
But in this case, you are doing an operation on `each` records, should be better to use `find_each`.